prisma/schema.prisma

```diff
@@ -0,0 +1,54 @@
+generator client {
+    provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Account {
```

Comment:
why do you need so many fields?

Reply:
most of them are default ones. https://authjs.dev/reference/adapters#models
some of them can be used later, like image

Reply:
image takes your google image and uses it with auth